src/IonParserTextRaw.ts

```diff
@@ -157,7 +157,7 @@ function get_type_from_name(str: string) : number {
   throw new Error("Unknown type: " + str + ".");
 }
 
-function is_keyword(str: string) : boolean {
+export function is_keyword(str: string) : boolean {
```

Comment:
Think it's best to extract this function into a component that's used by both `IonParserTextRaw` and other places that need to figure out if a string is a keyword instead of exposing it here to avoid weird coupling between `IonParserTextRaw` and other classes

Reply:
its dependent on a buttload of constants defined in the parser, pulling all of them out felt like a worse option.